ccm_web/client/src/api.ts

```diff
@@ -151,3 +155,36 @@ export const unmergeSections = async (sectionsToUnmerge: CanvasCourseSection[]):
   await handleErrors(resp)
   return await resp.json()
 }
+
+export const checkIfUserExists = async (loginId: string): Promise<boolean> => {
+  const request = getGet()
+  const resp = await fetch(`/api/admin/user/${loginId}`, request)
+  try {
+    await handleErrors(resp)
+  } catch (error: unknown) {
+    if (
+      error instanceof CanvasError &&
+      error.errors.length > 0 &&
+      error.errors[0].canvasStatusCode === 404
+    ) {
+      return false
+    } else {
+      throw error
+    }
```

Comment:
I thought about just examining the `resp` object first so this function didn't need to know about `CanvasError`, but I guess this ensures the 404 originates from Canvas and not our application. I'd be interested to hear others thoughts on this.

Reply:
@ssciolla what you did seem ok to me. I don't have anymore input on that.

Reply:
I think I see what you mean.  It _might_ be unnecessary, but in the end, I think being specific usually pays off.